apps/style/common.scss

```diff
@@ -1354,7 +1354,7 @@ $workspace-header-button-margin: ($workspace-headers-height -
 .invisible {
   visibility: hidden;
 }
-.hide-source {
+.hide-instructions {
```

Comment:
This name is still confusing. I can't remember why setting visualization to `inline-block` is done, but it impacts the flow of the doc on the left (differently when the responsive styles are on). Your change doesn't break anything (it is functionally equivalent for existing scenarios), but it isn't entirely clear whether you want this `inline-block` style or not for the new `hideSource` mode. I wish we had put a comment in the CSS file to explain why this change was made.